torch/generator.py

```diff
@@ -0,0 +1,106 @@
+import torch
+
+
+class Generator(torch._C._GeneratorBase):
```

Comment:
All this class does is forward to methods on the parent. If so, is there any point in subclassing this way? If it's docs you want, use something like `add_docstr`